src/sr_od/app/trailblaze_power/trailblaze_power_config.py

```
@@ -203,21 +203,49 @@ def loop(self, new_value: bool):
+
+        # 直接比较ID
+        return x.mission_id == y.mission_id
+
+    def get_next_plan(self, last_tried_plan: Optional[TrailblazePowerPlanItem] = None) -> Optional[
+        TrailblazePowerPlanItem]:
         """
-        按规划配置列表，找到第一个还没有完成的去执行
-        如果都完成了 选择第一个
-        :return: 下一个需要执行的计划
+        获取下一个未完成的计划任务。
+        如果提供了 last_tried_plan，则从该任务之后开始查找。
```

Ruff (0.13.1)

Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)